gradle.properties

```diff
@@ -30,7 +30,7 @@ spark13Version = 1.6.2
 spark20Version = 2.3.0
 spark22Version = 2.2.3
 spark24Version = 2.4.4
-spark30Version = 3.0.1
+spark32Version = 3.2.0
```

Comment:
This property rename seems to have broken the build (I think there were other places it was referenced. I'm not sure it's worth it. I changed these back locally, and everything seems to work for me.

Reply:
I reverted it - I think keeping it `30` is better for tracking earliest compatibility in the 3.x line for this version